File: Java&Kotlin lecture/src/main/java/com/java/many/Map.java
package com.java.many;

import java.util.HashMap;

public class Map {
    public static void main(String[] args) {
        HashMap<Integer, String> hashMap = new HashMap();
        hashMap.put(0, "A");
        hashMap.put(2, "B");
        hashMap.put(3, "C");
        hashMap.put(5, "D");
        hashMap.put(2, "E");

        System.out.println(hashMap);

        for(int key: hashMap.keySet()) {
            System.out.println(hashMap.get(key));
        }
    }
}

File: Java&Kotlin lecture/src/main/kotlin/com/kotlin/many/Map.kt
package com.kotlin.many

import java.util.*

fun main(args: Array<String>) {
    val hashMap: HashMap<Int, String> = HashMap()
    hashMap[0] = "A"
    hashMap[2] = "B"
    hashMap[3] = "C"
    hashMap[5] = "D"
    hashMap[2] = "E"

    println(hashMap)

    for (key in hashMap.keys) {
        println(hashMap[key])
    }

    for(value in hashMap.values) {
        println(value)
    }
}